### PR TITLE
Improving documentation related to Live Message feature

### PR DESCRIPTION
**Pull request by inkhey**
related to #3053 

*Please indicate here a summary of the pull request content. If a related issue exists, please indicate issue reference here*

## Checkpoints

The goal of this section is to help code reviewers and the merge request author to do the required checks on this pull request. Please don't edit this list.
If one or more of these checkpoints are out of scope, please write below why they are.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, then there are associated issues and issue is referenced in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present
- [x] Automated tests have been written (covering the feature or non regression), or *at least* an issue has been created to implement the test

**For developers**

- [x] Manual tests done to ensure stability on whole application layers, issue expectation follow
- [x] The original issue has been updated with a short summary of the implemented solution

**For the quality team**

- [x] Tested by the quality team

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -138,12 +164,18 @@ You can run some Tracim daemons too if you want those features:
...
-[http://127.0.0.1:6543](http://127.0.0.1:6543) and login with the admin account:
+You can now head to (if pushpin is correctly configured and use default port 7999)
+[http://127.0.0.1:7999](http://127.0.0.1:7999) and login with the admin account:
 
  * user: `admin@admin.admin`
  * password: `admin@admin.admin`
 
+:warning: If this does not work, you can try first:
+[http://127.0.0.1:6543](http://127.0.0.1:6543) and if it's working.
+It's a pushpin configuration issue.
```
Comment (raphj) [inline README.md:175]:
```suggestion
:warning: If this does not work, you can try to access [http://127.0.0.1:6543](http://127.0.0.1:6543). If it works, the issue is related to the configuration of pushpin.
```

File: backend/README.md [hunk 2]
```diff
@@ -209,7 +209,7 @@ in` development.ini.sample`
 :warning: By default, python warning are disabled. To enable warning please set
 `PYTHONWARNINGS` env var, for example `export PYTHONWARNINGS=default` .
 
-Run the Tracim backend web API:
+Run the Tracim backend web API (don't include live messages):
```
Comment (raphj) [inline backend/README.md:212]:
```suggestion
Run the Tracim backend web API (doesn't include live messages):
```

Comment (grignards) [inline backend/README.md:212]:
```suggestion
Run the Tracim backend web API (doesn't include live messages):
```

File: backend/README.md [hunk 3]
```diff
@@ -297,6 +298,31 @@ Run with (supervisord.conf should be provided, see [supervisord.conf default_pat
...
+    apt install pushpin
+
+and configure it to proxy tracim web (by default tracim web run on port 6543):
+
+    echo "* localhost:6543" > /etc/pushpin/routes
+
+and after this change you need to restart your pushpin service:
+
+    systemctl restart pushpin
+
```
Comment (grignards) [inline backend/README.md:325]: This is duplicated from the main README, is it necessary here ?

File: README.md [hunk 4]
```diff
@@ -102,6 +102,32 @@ Alternatively, under root:
...
+## For development purpose
+
+If you are in development environnement, the recommended way is to use docker,
+you should follow this documentation : [Live message setup for dev env](/backend/doc/live_message_setup.md)
+
+## For production usage
+
+You should install pushpin, for example in debian/ubuntu:
+
+    apt install pushpin
```
Comment (raphj) [inline README.md:119]:
```suggestion
    sudo apt install pushpin
```

File: README.md [hunk 5]
```diff
@@ -102,6 +102,32 @@ Alternatively, under root:
...
+
+and configure it to proxy tracim (by default tracim web run on port 6543):
+
+    echo "* localhost:6543" > /etc/pushpin/routes
```
Comment (raphj) [inline README.md:123]:
```suggestion
    echo "* localhost:6543" | sudo tee /etc/pushpin/routes
```

File: README.md [hunk 6]
```diff
@@ -102,6 +102,32 @@ Alternatively, under root:
...
+
+and after this change you need to restart your pushpin service:
+
+    systemctl restart pushpin
```
Comment (raphj) [inline README.md:127]:
```suggestion
    sudo systemctl restart pushpin
```

File: backend/README.md (hunk 3 above)
Comment (inkhey) [reply backend/README.md:325]: Normally this should be more detailled here as this is the specific backend documentation. I'm not sure how to organize to make both backend doc and defaut readme usable.

File: backend/README.md (hunk 2 above)
Comment (inkhey) [reply backend/README.md]: done (similar suggestion of @grignards )

**Comment by inkhey** [comment]
need to add :
`ProxyPassReverse /api/users/(.*/live_messages)$ http://127.0.0.1:7999/api/users/$1`

**Comment by inkhey** [comment]
When tracim 3.0 will be functionnally ok, this documentation need to be tested in many context, which may include:
- from scratch with script install in dev case.
- from scratch with script install in prod like case.
- from scratch with docker image.
- update from tracim v2 with docker image instance (in order to adapt documentation to simplify migration).
- update from tracim v2 with script install in "prod-like" case.

File: README.md [hunk 7]
```diff
@@ -102,6 +102,32 @@ Alternatively, under root:
 
 You can add "-d" to build_full_frontend.sh to disable obfuscation and reduce build time.
 
+### Install The Pushpin-Proxy for Live Messages
+
+Tracim rely on pushpin for live_message feature, so you need to have a working pushpin in order to make tracim work as
```
Comment (grignards) [inline README.md:107]:
```suggestion
Tracim relies on [pushpin](https://pushpin.org/docs/about/) for UI updates, so you need to setup pushpin.
```

File: README.md [hunk 8]
```diff
@@ -102,6 +102,32 @@ Alternatively, under root:
 
 You can add "-d" to build_full_frontend.sh to disable obfuscation and reduce build time.
 
+### Install The Pushpin-Proxy for Live Messages
```
Comment (grignards) [inline README.md:105]:
```suggestion
### Install and run pushpin for UI updates
```

**Comment by raphj** [comment]
I just saw a typo in the documentation on live messages:

```
PUSHPIN_CONFIG_DIR=./pushpin_cypress_config docker compose up -d pushpin
```

There should be a dash between docker and compose there. Can you add it?
I can also do it if you tell me to.

**Comment by raphj** [comment]
Another typo:

"pushping container" should read "pushpin container" in this file

File: README.md [hunk 9]
```diff
@@ -102,6 +102,32 @@ Alternatively, under root:
 
 You can add "-d" to build_full_frontend.sh to disable obfuscation and reduce build time.
 
+### Install and run pushpin for UI updates
+
+Tracim relies on [pushpin](https://pushpin.org/docs/about/) for UI updates, so you need to setup pushpin.
+expected.
```
Comment (grignards) [inline README.md:108]: I think we can remove this line.

Comment (inkhey) [reply README.md:108]:
```suggestion
```

**Pull request merged**